src/Compilers/VisualBasic/Portable/Syntax/VisualBasicSyntaxDiffer.vb

```diff
@@ -0,0 +1,27 @@
+﻿Namespace Microsoft.CodeAnalysis.VisualBasic
+    Friend NotInheritable Class VisualBasicSyntaxDiffer
+        Inherits SyntaxDiffer
+
+        Public Sub New(oldNode As SyntaxNode, newNode As SyntaxNode, computeNewText As Boolean)
+            MyBase.New(oldNode, newNode, computeNewText)
+        End Sub
+
+        Protected Overrides Function AreSimilarCore(node1 As SyntaxNode, node2 As SyntaxNode) As Boolean
+            Dim kindGroup =
```

Comment:
why are we allocating a function for this?  can we not just put this in a static helper?